### PR TITLE
New version: IrfanSkiljan.IrfanView.PlugIns version 4.66

### PR DESCRIPTION
**Pull request by CennoxX**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132931)

### CODE REVIEW

File: manifests/i/IrfanSkiljan/IrfanView/PlugIns/4.66/IrfanSkiljan.IrfanView.PlugIns.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: IrfanSkiljan.IrfanView
+Installers:
+- Architecture: x86
+  InstallerUrl: https://files03.tchspt.com/down/iview466_plugins_x64_setup.exe
+  InstallerSha256: 080FD4E30F5CDE44B52E85425E3E406F1BBBD9FF0226DCEC7D97BE813AAC84AB
+- Architecture: x64
+  InstallerUrl: https://files03.tchspt.com/down/iview466_plugins_setup.exe
+  InstallerSha256: C779C0C8307F90CDD798377C516C286ACDE6B9843AD84ED07B19D615248B605A
```
Comment (Okeanos) [inline manifests/i/IrfanSkiljan/IrfanView/PlugIns/4.66/IrfanSkiljan.IrfanView.PlugIns.installer.yaml:20]: The installer URLs look mixed up?

Comment (CennoxX) [reply manifests/i/IrfanSkiljan/IrfanView/PlugIns/4.66/IrfanSkiljan.IrfanView.PlugIns.installer.yaml]: You're right, I've now corrected the URLs.

**Comment by CennoxX** [comment]
These are official download links recommended by the author of IrfanView, taken from here: https://www.irfanview.com/download_sites.htm then followed the link [TechSpot - download IrfanView](https://www.techspot.com/downloads/299-irfanview.html) as the author doesn't provide a download directly from his site.

**Comment by stephengillie** [comment]
> These are official download links recommended by the author of IrfanView, taken from here: https://www.irfanview.com/download_sites.htm then followed the link [TechSpot - download IrfanView](https://www.techspot.com/downloads/299-irfanview.html) as the author doesn't provide a download directly from his site.

Thanks for the guide, as this developer has many mirrors.

**Comment by stephengillie** [comment]
This manifest contains a term that is blocked:

> True True

(Automated response - build 530.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**